src/SldStyleParser.ts

```diff
@@ -398,7 +401,7 @@ export class SldStyleParser implements StyleParser<string> {
     if (!operator) {
       return;
     }
-    const filter = this.getFilterFromOperatorAndComparison(operator, sldFilter);
+    const filter = this.getFilterFromOperatorAndComparison(operator as keyof typeof COMPARISON_MAP, sldFilter);
```

Comment:
Any chance to simplify `operator as keyof typeof COMPARISON_MAP`?
Maybe u can type the `SldOperator`s and then use `Record<SldOperator, Operator>` or something similiar.

Reply:
Extracted it out into a type.